rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -867,37 +910,42 @@ public void onSuccess(String result) {
             JSONObject p = arr.get(i).isObject();
 
             String pluginName = p.get("pluginName").isString().stringValue();
-            String pluginDescription = p.get("pluginDescription").isString().stringValue();
-            PluginDescriptor desc = new PluginDescriptor(pluginName, pluginDescription);
+            PluginDescriptor desc = getPluginDescriptor(p, pluginName);
 
-            JSONArray fields = p.get("configurableFields").isArray();
-            for (int j = 0; j < fields.size(); j++) {
-                JSONObject field = fields.get(j).isObject();
+            plugins.put(pluginName, desc);
+        }
 
-                String name = field.get("name").isString().stringValue();
-                String value = field.get("value").isString().stringValue();
+        return plugins;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)